.rhdh/docs/openshift.md

```diff
@@ -0,0 +1,256 @@
+## Using the Operator to manage Red Hat Developer Hub (RHDH)
```

Comment:
Can you switch this file to asciidoc? Downstream processes require .adoc as input to Pantheon.